README.md

```diff
@@ -1 +1,39 @@
-# Git Assignment - Ygong63
\ No newline at end of file
+# Git Assignment - Ygong63
+a. What is an issue?
+GitHub Issues is a communication method to mark and discuss with others
+
+b. What is a pull request?
+merge a set of changes from one branch into another
+
+c. How do I open up a pull request?
+git pull
```

Comment:
no

Reply:
On the GitHub.com, navigate to the main page of the repository. In the "Branch" menu, choose the branch that contains mycommits. Once I push the changes to my repo, the Compare & pull request button will appear in GitHub. A pull request can be opened by clicking the Create pull request button. 

git request-pull [-p] <start> <URL> [<end>]

Is this correct?

Reply:
Yes, personally I have only tried method 1, but if git request-pull works for you feel free to use it.